egov-persister/hearing-booking-persister.yml

```
@@ -99,4 +101,6 @@ serviceMaps:
 
             - jsonPath: $.Hearing.*.rescheduleRequestId
 
+            - jsonPath: $.Hearing.*.expiryTime
+              
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)